pkg/config/config.go

```diff
@@ -1327,3 +1322,28 @@ func (a *auth) ResolveJWKS(ctx context.Context) (string, error) {
 
 	return string(jwksEncoded), nil
 }
+
+// Retrieve the final base config to use taking into account the remotes override
+func (c *config) GetRemoteOverride(project_ref string) (overrideConfig baseConfig, overrideExist bool) {
```

Comment:
IMO this should throw an error if projectRef is not found.

```suggestion
func (c *config) GetRemoteOverride(projectRef string) (baseConfig, error) {
```